vendor/modules.txt

```diff
@@ -326,7 +326,6 @@ go.opentelemetry.io/otel/internal/baggage
 go.opentelemetry.io/otel/internal/global
 go.opentelemetry.io/otel/propagation
 go.opentelemetry.io/otel/semconv/v1.20.0
```

Comment:
Looks like the remaining users of v1.20;

- `instrumentation/net/http/otelhttp`; updating in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5923, but  not  yet  released?
- containerd? (not a dendency of  docker/cli)